sample-app/src/main/java/com/joinforage/android/example/ui/pos/POSComposeApp.kt

```diff
@@ -66,7 +70,12 @@ enum class POSScreen(@StringRes val title: Int) {
     PAYManualPANEntryScreen(title = R.string.title_pos_payment_manual_pan_entry),
     PAYMagSwipePANEntryScreen(title = R.string.title_pos_payment_swipe_card_entry),
     PAYPINEntryScreen(title = R.string.title_pos_payment_pin_entry),
-    PAYResultScreen(title = R.string.title_pos_payment_receipt)
+    PAYResultScreen(title = R.string.title_pos_payment_receipt),
+    VOIDTransactionTypeSelectionScreen(title = R.string.title_pos_void_flow),
+    VOIDPaymentScreen(title = R.string.title_pos_void_flow),
+    VOIDRefundScreen(title = R.string.title_pos_void_flow),
+    VOIDPaymentResultScreen(title = R.string.title_pos_void_flow),
+    VOIDRefundResultScreen(title = R.string.title_pos_void_flow)
```

Comment:
at some point we may want to distinguish the titles between payments and refunds. I just want to make sure that if we're testing, that it's abundantly clear to the user what test flow they're in. But, let's rack up some mileage with the current UI before I start speculating.